redfish-core/lib/fabric_adapters.hpp

```diff
@@ -221,6 +292,19 @@ inline void doAdapterGet(const std::shared_ptr<bmcweb::AsyncResp>& aResp,
     aResp->res.jsonValue["Status"]["State"] = "Enabled";
     aResp->res.jsonValue["Status"]["Health"] = "OK";
 
+    // add pcieslots
+    getFabricAdapterPCIeSlots(
+        aResp, systemName, fabricAdapterPath,
+        [aResp](const std::string& chassisName,
+                const dbus::utility::MapperEndPoints& /*pcieSlotPaths*/) {
```

Comment:
What is the purpose of returning pcieSlotPaths? Are we planning to use it in the future?

Reply:
This is to show the uri `/redfish/v1/Chassis/chassis<N>/PCIeSlots` only if the associated PCIeSlots are available.
As an example, Ed made similar comment in upstream https://gerrit.openbmc.org/c/openbmc/bmcweb/+/61097/comment/8740b2e5_d8a49a52/ like
```
All adapters have ports? That doesn't seem right.
```

In other words, there is no need to have the link to PCIeSlots unless there are PCIeSlots for it.